metricbeat/module/kubernetes/apiserver/apiserver.go

```diff
@@ -25,8 +25,30 @@ import (
 func init() {
 	mapping := &prometheus.MetricsMapping{
 		Metrics: map[string]prometheus.MetricMap{
-			"apiserver_request_count":     prometheus.Metric("request.count"),
-			"apiserver_request_latencies": prometheus.Metric("request.latency"),
+
+			// Process metrics
+			"process_cpu_seconds_total":     prometheus.Metric("process.cpu.sec"),
+			"process_resident_memory_bytes": prometheus.Metric("process.memory.resident.bytes"),
+			"process_virtual_memory_bytes":  prometheus.Metric("process.memory.virtual.bytes"),
+			"process_open_fds":              prometheus.Metric("process.fds.open.count"),
+			"process_start_time_seconds":    prometheus.Metric("process.started.sec"),
+
+			// HTTP server metrics
+			"http_request_duration_microseconds": prometheus.Metric("http.request.duration.us"),
+			"http_request_size_bytes":            prometheus.Metric("http.request.size.bytes"),
+			"http_response_size_bytes":           prometheus.Metric("http.response.size.bytes"),
+			"http_requests_total":                prometheus.Metric("http.request.count"),
+
+			// REST metrics
+			"rest_client_requests_total": prometheus.Metric("client.request.count"),
+
+			"apiserver_request_latencies":             prometheus.Metric("request.latency"),
+			"apiserver_request_total":                 prometheus.Metric("request.count"),
```

Comment:
This metric was introduced in 1.14, till 1.13 `apiserver_request_count` should be used. Is there a way we can do to use one if available and the other one if not?
If we cannot do it I am not sure what we should do, this would be a breaking change for all clusters < 1.14, and in 1.15 `apiserver_request_count` still exists. Maybe we should keep the old metric by now, that is compatible with all released versions, though ideally we should support both metrics.

Would this work? (it may work as expected if `apiserver_request_total` always appear after `apiserver_request_count`, so the new metric overwrites the old one).
```suggestion
	"apiserver_request_total":                 prometheus.Metric("request.count"),
	"apiserver_request_count":                 prometheus.Metric("request.count"),
```

Maybe for this kind of cases we should add an option to decide what to do if a field is already set:
```suggestion
	"apiserver_request_total":    prometheus.Metric("request.count", prometheus.OpOverwrite(true)),
	"apiserver_request_count":    prometheus.Metric("request.count", prometheus.OpOverwrite(false)),
```